app/lib/actions/katello/host/erratum/install.rb

```diff
@@ -9,10 +9,20 @@ def plan(host, errata_ids)
             Type! host, ::Host::Managed
 
             action_subject(host, :errata => errata_ids)
-            plan_action(Pulp::Consumer::ContentInstall,
-                        consumer_uuid: host.content_facet.uuid,
-                        type:          'erratum',
-                        args:          errata_ids)
+            # Batch install errata in 6.2.z for BZ1288152
+            if Setting['erratum_install_batch_size'] && Setting['erratum_install_batch_size'] > 0
+              errata_ids.each_slice(Setting['erratum_install_batch_size']) do |errata_ids_batch|
```

Comment:
@dLobatog It looks the same in the foreman_tasks UI, but you can notice the difference in Dynflow. Here's 40 errata installed in 2 batches:

![image](https://cloud.githubusercontent.com/assets/761923/22347428/f2183b2c-e3d5-11e6-8c11-4d66ea3ca63e.png)